proofs/cbmc/store64/store64_harness.c

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2025 The mldsa-native project authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "fips202/fips202.h"
+
+extern void store64(uint8_t x[8], const uint64_t u);
```

Comment:
As mentioned above, the `const` has no meaning for external signature because `u` is passed by value.